src/app/mat-select-search/mat-select-search.component.ts

```diff
@@ -285,6 +285,30 @@ export class MatSelectSearchComponent implements OnInit, OnDestroy, AfterViewIni
             this.changeDetectorRef.markForCheck();
           });
       });
+
+      if (this.matSelect._keyManager) {
```

Comment:
Unfortunately, it looks like at this point, _keyManager was sometimes undefined. (It was the case for the app-custom-clear-icon-example component)

Is there a safer place I could subscribe to the event, where I can know for sure that _keyManager is defined?

Reply:
it is initialized in the `ngAfterContentInit` lifecycle hook:
https://github.com/angular/material2/blob/eb5eed14778f402ea56a096148ff6e78d8bdad93/src/lib/select/select.ts#L542-L543

Reply:
so we need to try to initialize it in the `ngAfterViewInit()` and in the `ngAfterContentInit` lifecycle hook

Reply:
we should initialize first time after the mat select is opened (See https://github.com/bithost-gmbh/ngx-mat-select-search/pull/54):
https://github.com/bithost-gmbh/ngx-mat-select-search/blob/4b92230643c50a777be5128b1fea014e9abace43/src/app/mat-select-search/mat-select-search.component.ts#L226-L232
however, i would keep the `if (this.matSelect._keyManager) {` and maybe add a console.log() to avoid errors if in some circumstances the keymanager is not initialized yet. See e.g. https://github.com/bithost-gmbh/ngx-mat-select-search/issues/53

Reply:
I moved the subscribtion and added the console.log. 

However, I added a placeholder message for now, let me know what you think it should include.